templates/repo/tag/list.tmpl

```diff
@@ -0,0 +1,13 @@
+{{/* This template is only used to show tags list when repo's release setting is disabled. */}}
```

Comment:
We need a test for this.

Otherwise, the bugs in this template won't be found. Because most users/developers do not disable this unit, the bugs would be undiscoverable and would go into stable releases.

Reply:
Reused the old `releasesOrTags` function

Reply:
Then we can also put `/repo/tag/table.tmpl` back? 🤣

Reply:
🤔 Reusing `repo/release/list.tmpl` is feasible technically, but I'm afraid it will make the code too complicated. I'll try to simplify the code and reuse it.

Reply:
Reused the template in https://github.com/go-gitea/gitea/pull/23465/commits/c76333f7fbbea4d9705a2e5945568b2c891329e9